src/js/tabs/failsafe.js

```diff
@@ -265,8 +268,13 @@ failsafe.initialize = function (callback) {
         $('input[name="gps_rescue_throttle_max"]').val(FC.GPS_RESCUE.throttleMax);
         $('input[name="gps_rescue_throttle_hover"]').val(FC.GPS_RESCUE.throttleHover);
         $('input[name="gps_rescue_min_sats"]').val(FC.GPS_RESCUE.minSats);
+        // Sort the element, if need to group, do it by lexical sort, ie. by naming of (the translated) selection text
+        $('#failsafeGpsRescueItemSanitySelect').sortSelect();
```

Comment:
Could we follow similar patter to above of using `name` instead of `id`? so we don't need extra property to keep in sync and this would look like:
```suggestion
        $('select[name"gps_rescue_sanity_checks"]').sortSelect();
```